src/post_processing/post_proc_generator.jl

```diff
@@ -1194,6 +1194,7 @@ function _mechanical_torque(
     name::String,
     res::SimulationResults,
     dt::Union{Nothing, Float64, Vector{Float64}},
+    unique_timestamps::Bool,
```

Comment:
include a default 

```suggestion
    unique_timestamps::Bool = true
```

So that the tests won't break.